config/config.exs

```diff
@@ -1,4 +1,4 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
-import_config "#{Mix.env}.exs"
+import_config "deploy/#{Mix.env}.exs"
```

Comment:
Shouldn't this line just be removed? At least it seems a bit confusing to import "deploy" config into the Bootleg config 🤔

Reply:
Yes, good catch @pmeinhardt 👍